test/zz_neo4j_pg_test.py

```diff
@@ -301,12 +301,11 @@ def test_3_site_load(self):
         ls = set()
         for l in links:
             ls.add(l[1])
-        assert('node_id-00-00-00-00-00-11-Wave' in ls and
-               'node_id-10-00-00-00-00-11-Wave' in ls and
-               'node_id-00-00-00-00-00-10-Wave' in ls)
+        assert('port+renc-data-sw:HundredGigE0/0/0/26-Wave' in ls and
```

Comment:
Just a comment and XI may have to validate this but i noticed the name of the ports on the dataplane can change according to the speed of the port.

Reply:
This is the scheme that @xi-yang uses in his network am ad, so I'm just trying to conform to that. I guess the question is how often does the speed change?

Reply:
i am not entirely sure about it. But this was one of the reason not to adding these interfaces into ralph via auto-detect

Reply:
I think we don't need the switch interfaces in Ralph per se. What is important is that server interfaces have information about the switch interfaces they are connected to.